dashboard/app/controllers/api_controller.rb

```diff
@@ -153,6 +153,105 @@ def section_text_responses
     render json: data
   end
 
+  def section_assessments
+    load_section
+    load_script
+
+    level_group_script_levels = @script.script_levels.includes(:level).where('levels.type' => LevelGroup)
+
+    data = @section.students.map do |student|
+      student_hash = {id: student.id, name: student.name}
+
+      level_group_script_levels.map do |script_level|
+        next unless script_level.long_assessment?
```

Comment:
I don't think we have one right now, but it seems possibly, and at least for logic like this I like the idea of them being independent.